sktime/tests/test_all_estimators.py

```diff
@@ -894,6 +895,8 @@ def test_inheritance(self, estimator_class):
                 f"estimator: {estimator_class} has fit method, but"
                 f"is not a sub-class of BaseEstimator."
             )
+        if isinstance(Pipeline(), estimator_class):
```

Comment:
@fkiraly, any ideas on how to fix this test in a prettier way? 

The problem is that the Pipeline inherits from BaseEstimator but not from a more specific class as BaseForecaster, etc. This results in the failing of the assertion about the number of base classes.

Reply:
well, this works for now. We should later revisit the test.